config/checkstyle-non-main-files-suppressions.xml

```diff
@@ -67,6 +67,8 @@
          lines="1"/>
   <suppress id="lineLength"
          files="src[\\/]it[\\/]resources[\\/].*[\\/]InputNoLineWrapping\.java"/>
+  <suppress id="lineLength"
+         files="src[\\/]it[\\/]resources[\\/].*[\\/]rule4861blockcommentstyle[\\/].*"/>
```

Comment:
added this suppression because the message was too long. It was crossing the column limit even when I tried to trim the message with `.*` and used `violation above`.

Reply:
not good reason, let me try to suggest something.

Reply:
@Zopsss , please remove this suppression